### PR TITLE
Feature/implement input

### PR DESCRIPTION
**Pull request by ericlp**
This PR should enable basic input based on Libgdx input API.

### CODE REVIEW

File: core/src/main/se/tevej/game/input/TKeyBoard.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+public interface TKeyBoard {
+
+    TKeyBoard addClickedListener(OnClickedListener onClickedListener);
+
+    interface OnClickedListener {
+        void onClicked (TKeyBoard keyBoard, Key button);
+    }
+
+    enum Key {
```
Comment (ViddeM) [inline core/src/main/se/tevej/game/input/TKeyBoard.java]: This seems unnecessary, this already exits in Input.Keys.

File: core/src/main/se/tevej/game/input/TKeyBoard.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package main.se.tevej.game.input;
+
+import static com.badlogic.gdx.Input.Keys.*;
+
+public interface TKeyBoard {
+
+    TKeyBoard addClickedListener(OnClickedListener onClickedListener);
+
+    interface OnClickedListener {
```
Comment (ViddeM) [inline core/src/main/se/tevej/game/input/TKeyBoard.java]: Why does an interface have an inner interface? If there does not exist an extremely good reason, please change.

File: core/src/main/se/tevej/game/input/TMouse.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    TMouse addClickedListener(OnClickedListener onClickedListener);
+
+    TMouse addMovedListener(OnMovedListener onMovedListener);
+
+    float getX();
+
+    float getY();
+
+    interface OnDraggedListener {
+        void onDragged(TMouse mouse, MouseButton button, float x, float y );
```
Comment (ViddeM) [inline core/src/main/se/tevej/game/input/TMouse.java]: All of these inner interfaces should probably not exist (Same as earlier comment).

Comment (ericlp) [reply core/src/main/se/tevej/game/input/TMouse.java]: I am not sure as of now and want to hear from both your side and portals side regarding this issue. I want to merge this pull-request and i have added an issue to fix the inner classes.

**Pull request closed without merging**